docs/esp32/quickref.rst

```diff
@@ -240,6 +240,73 @@ ESP32 specific ADC class method reference:
       - ``ADC.WIDTH_11BIT``: 11 bit data
       - ``ADC.WIDTH_12BIT``: 12 bit data - this is the default configuration
 
+DAC (digital to analog conversion)
+----------------------------------
+
+On the ESP32 DAC functionality is available on Pins 25 and 26. Note that, when creating
+the DAC object for a pin, the DAC value will be initialized at 0.
+
+Use the :ref:`machine.DAC <machine.DAC>` class::
+
+    from machine import DAC
+
+    dac = DAC(Pin(25))          # create DAC object on pin 25
+    dac.write(128)              # set the output voltage to 1.65V
+
+    dac.cosine_enable()         # enable the cosine generator
+    dac.frequency_set(7, 1)     # set the output frequency using the RTC clock divider and 
+                                # frequency steps for the CW generator, (7, 1) -> ~15 Hz
+    
+
+ESP32 specific DAC class method reference:
```

Comment:
This doc is just the quick reference. You should place the detailed docs into docs/library/esp32.rst - IMHO

Reply:
Hmm, I mirrored what the ADC does? Should that be moved too?